qiskit/circuit/quantumcircuit.py

```diff
@@ -299,6 +300,23 @@ def data(self) -> QuantumCircuitData:
         """
         return QuantumCircuitData(self)
 
+    @property
+    def op_start_times(self) -> List[int]:
+        """Return a list of operation start times.
+
+        This attribute is enabled once one of scheduling analysis passes
+        runs on the quantum circuit.
+
+        Returns:
+            List of integers representing instruction start times.
+            The index corresponds to the index of instruction in :attr:`QuantumCircuit.data`.
+        """
+        if not self._op_start_times:
```

Comment:
I think we should differentiate between an empty list and None. What if someone does:

```python
transpile(QuantumCircuit(2), backend=FakeMontreal(), scheduling_method='alap')
```
this would raise an `AttributeError` even though the circuit is scheduled.

```suggestion
        if self._op_start_times is None:
```